### PR TITLE
SCRUM-2394 asserted genes and notes updates

### PR DESCRIPTION
**Pull request by adamgibs**
-- asserted genes display symbol instead of curie
   -- main row gene is filtered out of asserted genes
   -- notes are prefixed with either note or summary depending on type

### CODE REVIEW

File: apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function AnnotatedEntitiesPopupCuration(props) {
                   <tr key={entity.subject.curie}>
                     <td>{renderLink(entity)}</td>
                     <td><TypeCellCuration subject={entity.subject}/></td>
-                    <td><AssertedGenes assertedGenes={entity.assertedGenes}/></td>
+                    <td><AssertedGenes assertedGenes={entity.assertedGenes?.filter(gene => gene.curie !== mainRowCurie)}/></td>
```
Comment (oblodgett) [inline apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js:84]: @adamgibs can this logic be moved into the component?

Comment (oblodgett) [reply apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js:84]: Send both assertedGenes and gene to the component and let it figure it out.

**Review by oblodgett — CHANGES_REQUESTED** [review]
@adamgibs can this logic be moved into the component?

**Comment by adamgibs** [comment]
@oblodgett logic moved into component

**Review by oblodgett — APPROVED** [review]

**Pull request merged**